docs/src/main/paradox/user/lang/dart/gettingstarted.md

```diff
@@ -0,0 +1,79 @@
+# Getting started with stateful services in Dart
+
+## Prerequisites
+
+### Dart version
+Cloudstate Dart support requires Dart sdk >=2.7.0 <3.0.0.
+
+### Build tool
+Use Dart [Pub](https://dart.dev/tools/pub/cmd) tool for build Cloudstate Dart projects.
+
+### Protoc
+Since Cloudstate is based on gRPC, you need a protoc compiler to compile gRPC protobuf descriptors. While this can be done by downloading, [installing and running protoc manually](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation). You will also need to add a plugin to compile the proto files for Dart classes.
+
+```shell script
+pub global activate protoc_plugin
+```
+
+The compiler plugin, protoc-gen-dart, is installed in $HOME/.pub-cache/bin. It must be in your PATH for the protocol compiler, protoc, to find it.
+
+```shell script
+export PATH=$PATH:$HOME/.pub-cache/bin
+```
+
+### Docker
+Cloudstate runs in Kubernetes with [Docker](https://www.docker.com/), hence you will need Docker to build a container that you can deploy to Kubernetes. Below is a Dockerfile file with Dart support:
+
+```shell script
+FROM google/dart
+
+WORKDIR /app
+
+ADD pubspec.yaml /app/
+RUN pub get
+ADD . /app
+RUN pub get --offline
+RUN dart --snapshot-kind=kernel --snapshot=bin/main.dart.snapshot bin/main.dart
+
+CMD []
+ENTRYPOINT ["/usr/bin/dart", "--enable-asserts",  "--enable-vm-service:8181", "bin/main.dart.snapshot"]
+```
+
+In addition to the above, you will need to install the Cloudstate Dart support library, which can be done as follows:
+
+: @@@vars   
```

Comment:
This is currently displaying as a tabbed definition list, because of the `: `

<img width="1073" alt="Screen Shot 2020-05-08 at 3 14 25 PM" src="https://user-images.githubusercontent.com/59895/81366391-a1fe0300-913e-11ea-8342-a837d81039b3.png">

Expect this should just be a regular paragraph and code block.

```suggestion
@@@vars
```